src/handler.js

```diff
@@ -114,11 +116,45 @@ const handle = async message => {
       Authorization: `Bearer ${process.env.ACCESS_TOKEN}`,
     },
   }).then(response => {
-    response.data.pipe(resizer).pipe(writeStream);
+    response.data.pipe(resizer).pipe(resizedStream);
+  });
+
+  let b64Chunks = [];
+
+  resizedStream.pipe(b64Stream);
+
+  b64Stream.pipe(new Base64Encode());
+
+  b64Stream.on('data', chunk => {
+    b64Chunks.push(chunk);
+  });
+  const [owner, repo] = process.env.GITHUB_REPO.split('/');
+  const branch = process.env.GITHUB_REPO_BRANCH;
+  const emojiRepoDir = process.env.GITHUB_REPO_DIR;
+  const emojiPath = path.join(emojiRepoDir, `${emojiAlias}.${filetype}`);
+
+  b64Stream.on('finish', async () => {
+    console.log('b64', Buffer.concat(b64Chunks).toString('base64'));
+    try {
+      await octokit.repos.createFile({
+        owner,
+        repo,
+        path: emojiPath,
+        branch,
+        message: `emojo: added :${emojiAlias}:`,
+        content: Buffer.concat(b64Chunks).toString('base64'),
```

Comment:
Whilst this will create a base64 encoded string, make sure the output has the file mimetype etc on the front of it or it won't be able to be rendered as an image properly.

Might be something the library you've used does to be fair but just double check that it has a mime looking prefix on the string.

```
const prefix = `data:image/gif;base64,`;
const b64 = Buffer.concat(chunks).toString('base64');
const combined = `${prefix}${b64}`;
```

for example